src/test-runs.ts

```diff
@@ -161,7 +94,7 @@ export class TestRuns {
 
             let tld = p.getTLD();
 
-            LanguageExtractor.isWebPageInLanguage(p, LanguageExtractor.ENGLISH_LANG_CODE, function(result : boolean) {
+            LanguageExtractor.isWebPageInLanguage(p, "en", function(err, result) {
```

Comment:
Will throw an Error. The parameter is standardized under ISO 639-1. Basically 2 character language codes.